### PR TITLE
Add ethics examination to research life cycle, fixes #261

### LINKED ISSUE #261: Add notion of research integrity into research lifecycle
Again about awareness,
we should add an awareness of research ethics 
and integrity as part of the research lifecycle.

### PR DESCRIPTION
**Pull request by jpthiele**

### CODE REVIEW

**Comment by jpthiele** [comment]
Related to #261

File: competencies.md [hunk 1]
```diff
@@ -526,8 +526,13 @@ One of the crucial skills of RSEs is their mental proximity to research.
...
 despite friendly competition, shares the common goal of gaining knowledge
 for its own sake and not just for personal or commercial gain.
-Thereby they know that they are part of a bigger cycle that involves many other parties in and outside
-their domain, and also that their software can be utilised at different stages of the research cycle by different people.
+Thereby they know that they are part of a bigger cycle 
+that involves many other parties in and outside their domain, 
+and also that their software can be utilised at different stages 
+of the research cycle by different people.
+They also know that the examination of ethical concerns 
+can be an important step in the life-cycle of a specific project
```
Comment (MakisH) [inline competencies.md:534]:
```suggestion
can be an important step in the life-cycle of a specific project,
```

**Review by MakisH — APPROVED** [review]

File: competencies.md [hunk 2]
```diff
@@ -526,8 +526,13 @@ One of the crucial skills of RSEs is their mental proximity to research.
...
 despite friendly competition, shares the common goal of gaining knowledge
 for its own sake and not just for personal or commercial gain.
-Thereby they know that they are part of a bigger cycle that involves many other parties in and outside
-their domain, and also that their software can be utilised at different stages of the research cycle by different people.
+Thereby they know that they are part of a bigger cycle 
+that involves many other parties in and outside their domain, 
+and also that their software can be utilised at different stages 
+of the research cycle by different people.
+They also know that the examination of ethical concerns 
+can be an important step in the life-cycle of a specific project,
```
Comment (CaptainSifff) [inline competencies.md:534]:
```suggestion
can be an important step in the life cycle of a specific project, 
```
since I just consistified that...

**Review by CaptainSifff — APPROVED** [review]

File: competencies.md [hunk 3]
```diff
@@ -526,8 +526,13 @@ One of the crucial skills of RSEs is their mental proximity to research.
 They embrace being part of a larger community which,
 despite friendly competition, shares the common goal of gaining knowledge
 for its own sake and not just for personal or commercial gain.
-Thereby they know that they are part of a bigger cycle that involves many other parties in and outside
-their domain, and also that their software can be utilised at different stages of the research cycle by different people.
+Thereby they know that they are part of a bigger cycle
```
Comment (mhagdorn) [inline competencies.md:529]: I think cycle isn't quite the right work here, what about 

they are part of a bigger group

File: competencies.md [hunk 4]
```diff
@@ -526,8 +526,13 @@ One of the crucial skills of RSEs is their mental proximity to research.
 They embrace being part of a larger community which,
 despite friendly competition, shares the common goal of gaining knowledge
 for its own sake and not just for personal or commercial gain.
-Thereby they know that they are part of a bigger cycle that involves many other parties in and outside
-their domain, and also that their software can be utilised at different stages of the research cycle by different people.
+Thereby they know that they are part of a bigger cycle 
+that involves many other parties in and outside their domain, 
+and also that their software can be utilised at different stages 
+of the research cycle by different people.
+They also know that the examination of ethical concerns
```
Comment (mhagdorn) [inline competencies.md:533]: maybe

They may be asked to contribute to the ethical evaluation of a project to ensure integrity....

this makes it clear that they should be able to take an active role in the ethical evaluation of a research project

**Review by mhagdorn — CHANGES_REQUESTED** [review]
a long standing quibble (that should get changed) and a minor reformulation suggestion

File: competencies.md (hunk 3 above)
Comment (jpthiele) [reply competencies.md:529]: Group feels somehow too vague for me, 
how about one of the following 
- (human) endeavour
- undertaking

Or something along those lines to emphasize the collective effort of humanity that research ultimately is.

Comment (mhagdorn) [reply competencies.md:529]: Admittedly I was thinking strictly group of people. Your suggestion is much better. _Endeavour_ sounds a bit too grand to me, _undertaking_ is good.

**Pull request merged**